src/styles/main.scss

```diff
@@ -1,3 +1,67 @@
+$start-days:
+  mon 0,
+  tue 1,
+  wed 2,
+  thu 3,
+  fri 4,
+  sut 5,
+  sun 6;
+$cell-size: 100px;
+$cell-color: #eee;
+$choosed-color: pink;
+$gap-size: 1px;
+
+html {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
 body {
-  margin: 0;
+  margin: 0 auto;
+}
+
+.calendar {
+  margin: 80px auto 0;
+  width: (($cell-size + $gap-size) * 7) - $gap-size;
```

Comment:
You can transfer mathematical operations to separate variables and use the result immediately, this will increase readability.